openstax-poland/adaptarr-front
# Tickets routing

https://github.com/openstax-poland/adaptarr-backlog/issues/44

### Testing
1. Go to helpdesk
2. Click <kbd>Create new ticket</kbd>
3. Url should change - refresh - everything should stay the same
4. Go back <kbd><</kbd> - url should change
5. Open ticket
6. Url should change - refresh - everything should stay the same
7. Change ticket id in url to something like `not-exists` and hit enter
8. Error message should be displayed
9. Search for notification with `New support ticket was created by XXX.`
10. Click on it - it should redirect you to specific ticket
